src/Symfony/Component/ErrorHandler/README.md

```diff
@@ -21,6 +21,9 @@ Debug::enable();
 //ErrorHandler::register();
 //DebugClassLoader::enable();
 
+// If you want a custom generic template when debug is not enabled
+// HtmlErrorRenderer::setTemplate('../path/to/custom/error.html.php');
```

Comment:
now the relative path '..` can be removed to avoid confusing people.

Reply:
Done.